skyportal/handlers/api/candidate/scan_report_item.py

```diff
@@ -0,0 +1,172 @@
+from astropy.time import Time
+
+from baselayer.app.access import auth_or_token
+from baselayer.log import make_log
+
+from ....models import Obj, Source
+from ....models.scan_report.scan_report_item import ScanReportItem
+from ....utils.safe_round import safe_round
+from ...base import BaseHandler
+
+log = make_log("api/scan_report_item")
+
+
+def create_scan_report_item(session, report, sources_by_obj):
+    """
+    Parameters
+    ----------
+    session: sqlalchemy.orm.Session
+    report: skyportal.model.ScanReport
+        The scan report to create an item for
+    sources_by_obj: tuple (obj_id, source_ids)
+        The object and link source ids to create the item for
+    Returns
+    -------
+    scan_report_item: skyportal.model.ScanReportItem
+    """
+    obj_id, source_ids = sources_by_obj
+
+    obj = session.scalar(
+        Obj.select(session.user_or_token, mode="read").where(Obj.id == obj_id)
+    )
+    current_mag = obj.photstats[0].last_detected_mag if obj.photstats else None
+    current_age = (
+        (Time.now().mjd - obj.photstats[0].first_detected_mjd)
+        if obj.photstats
+        else None
+    )
+
+    sources = session.scalars(
+        Source.select(session.user_or_token, mode="read").where(
+            Source.obj_id == obj_id, Source.id.in_(source_ids)
+        )
+    ).all()
+
+    return ScanReportItem(
+        obj_id=obj.id,
+        scan_report=report,
+        data={
+            "saved_infos": [
```

Comment:
`saved_info` maybe?

Reply:
from a frenchman to another, here's a tip: information doesn't have a plural in English